.github/workflows/android-beta.yaml

```diff
@@ -41,6 +41,7 @@ jobs:
         shell: bash
         run: |
           (cd tools/android; ./set_up_android.py --sdk $ANDROID_SDK_ROOT --ndk $ANDROID_NDK_ROOT)
+          (echo "yes" | $ANDROID_HOME/tools/bin/sdkmanager "cmake;3.18.1")
```

Comment:
I don't know how to explain this, but it works via Android Studio, but doesn't work via command-line.